### ISSUE TITLE
Modal close "icon" should be made larger

### ISSUE THREAD

**Issue by SteveALee** [netlify/netlify-identity-widget#349]
The `btnClose` is 24px x 24 px but the minimum recommend by WCAG is 44px x 44 px. This assures good touch device access.

https://www.w3.org/WAI/WCAG21/Understanding/target-size.html

It will also need to be moved so it doesn't clash with the 2nd tab.
